### ISSUE TITLE
Training from scratch

### ISSUE THREAD

**Issue by UdonDa** [fenglinglwb/MAT#20]
Hi, @fenglinglwb!
Congratulations on CVPR2022 Best Paper Finalists!!

I want to train a MAT from scratch with CelebAHQ 128x128. But, training cannot correctly be performed with lr=0.001.
Inputs images are normalized to [-1, 1] and 0 value in mask indicates a missing pixel.
I confirmed the input worked for your pretrained weights.

Did you see this problem?

Input images
![input_in_loop](https://user-images.githubusercontent.com/25411643/175813446-86f10cca-5a1d-4b1f-9b07-8e3d5420f05e.jpg)

Generated images in my training process.
<img width="498" alt="image" src="https://user-images.githubusercontent.com/25411643/175813462-95bfb795-f664-4075-a760-6687e879af4d.png">

**Comment by fenglinglwb** [comment]
The training seems to have failed. Have you checked the loss log? You may try to lower the learning rate and masking ratio or remove the style manipulation module to find the reason for failure.
